src/featbin/wav-copy.cc

```diff
@@ -26,35 +27,69 @@ int main(int argc, char *argv[]) {
   try {
     using namespace kaldi;
     const char *usage =
-        "Copy archives of wave files\n"
+        "Copy wave file or archives of wave files\n"
         "\n"
-        "Usage:  wav-copy [options...] <wav-rspecifier> <wav-rspecifier>\n"
+        "Usage: wav-copy [options] <wav-rspecifier> <wav-wspecifier>\n"
+        "  or:  wav-copy [options] <wav-rxfilename> [<wav-wxfilename>]\n"
         "e.g. wav-copy scp:wav.scp ark:-\n"
+        "     wav-copy wav.ark:123456 \n"
         "See also: wav-to-duration extract-segments\n";
-    
+
     ParseOptions po(usage);
 
     po.Read(argc, argv);
 
-    if (po.NumArgs() != 2) {
+    if (po.NumArgs() < 1 || po.NumArgs() > 2) {
```

Comment:
Yes, please squash.

On Sat, Aug 27, 2016 at 5:11 PM, Peter Smit <notifications@github.com>
wrote:

> In src/featbin/wav-copy.cc
> <https://github.com/kaldi-asr/kaldi/pull/1005#discussion_r76522440>:
>
> >      ParseOptions po(usage);
> >
> >      po.Read(argc, argv);
> >
> > -    if (po.NumArgs() != 2) {
> > +    if (po.NumArgs() < 1 || po.NumArgs() > 2) {
>
> Ok, I'll change this as well. Should I squash this complete PR into one
> new commit?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/1005/files/4a1da846be6bd3f2c595aa5cc71ebae008a36638#r76522440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADJVu-k2q0Mf1Adbwog5ckK38mQ4ZqG3ks5qkKfmgaJpZM4Jr51e>
> .
>